### ISSUE TITLE
how can i transform from the code in python "            nn.Conv2d(in_channels=1, out_channels=40, kernel_size=(3, 9), stride=(1, 3), padding=0, bias=False)" to libtorch with c++?

### ISSUE THREAD

**Issue by nsjiow** [pytorch/pytorch#17441]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by nsjiow** [comment]
training for rectangle image

**Comment by nsjiow** [comment]
btw, where can i find more libtorch docs?

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions
